infraohjelmointi_api/services/ProjectWiseService.py

```diff
@@ -102,7 +104,7 @@ def sync_project_from_pw(self, project: Project) -> None:
     def get_project_from_pw(self, id: str):
         """Method to fetch project from PW with given PW project id"""
         start_time = time.perf_counter()
-        api_url = f"{self.pw_api_url}{self.pw_api_project_endpoint}?${self.pw_api_project_endpoint_filter}{id}"
+        api_url = f"{self.pw_api_project_metadata_url}&${self.pw_api_project_metadata_endpoint}+{id}"
```

Comment:
concatenate id without + sign as it was before. Also & sign should be in env variable value